connect/runtime/src/test/java/org/apache/kafka/connect/runtime/errors/ErrorReporterTest.java

```diff
@@ -184,9 +187,20 @@ private ProcessingContext processingContext() {
         return context;
     }
 
-    private Map<String, Object> config(String key, Object val) {
-        config.put(key, val);
-        return config;
+    private SinkConnectorConfig sinkConfig(Map<String, String> configProps) {
+        Map<String, String> props = new HashMap<>();
+        props.put(ConnectorConfig.NAME_CONFIG, "test");
+        props.put(ConnectorConfig.CONNECTOR_CLASS_CONFIG, SinkTask.class.getName());
+        props.putAll(configProps);
+        return new SinkConnectorConfig(plugins, props);
+    }
+
+    private ConnectorConfig config(Map<String, String> configProps) {
+        Map<String, String> props = new HashMap<>();
+        props.put(ConnectorConfig.NAME_CONFIG, "test");
+        props.put(ConnectorConfig.CONNECTOR_CLASS_CONFIG, SinkTask.class.getName());
+        props.putAll(configProps);
+        return new ConnectorConfig(plugins, props);
```

Comment:
Since SinkConnectorConfig extends ConnectorConfig, could we just use the first helper method everywhere?